app/lib/report_helper.rb

```diff
@@ -195,7 +195,8 @@ def all_categories
 
   def fetch_taxon_counts(pipeline_run_id, background_id)
     pipeline_run = PipelineRun.find(pipeline_run_id)
-    adjusted_total_reads = pipeline_run.total_reads * pipeline_run.subsample_fraction
+    total_ercc_reads = pipeline_run.ercc_counts.map(&:count).sum # will be zero if no ercc_counts
```

Comment:
we should just put the total in a new pipeline_run field so we can avoid querying database every time for this.

Reply:
ok